src/js/carousel.js

```diff
@@ -109,54 +109,55 @@
           $this.attr('aria-describedby', id_desc);
           $this.prepend('<p  id="' + id_desc   + '" class="sr-only">A carousel is a rotating set of images, rotation stops on keyboard focus on carousel tab controls or hovering the mouse pointer over images.  Use the tabs or the previous and next buttons to change the displayed slide.</p>')
           $this.prepend('<h2 id="' + id_title  + '" class="sr-only">Carousel content with ' + $tabpanels.length + ' slides.</h2>')
-        }  
+        }
+
 
-                
         for (i = 0; i < $tabs.length; i++) {
           $tab = $tabs[i]
-          
+
           $tab.setAttribute('role', 'tab')
           $tab.setAttribute('id', 'tab-' + index + '-' + i)
           $tab.setAttribute('aria-controls', 'tabpanel-' + index + '-' + i)
-          
+
           var tpId = '#tabpanel-' + index + '-' + i
           var caption = $this.find(tpId).find('h1').text()
-          
+
           if ((typeof caption !== 'string') || (caption.length === 0)) caption = $this.find(tpId).text()
           if ((typeof caption !== 'string') || (caption.length === 0)) caption = $this.find(tpId).find('h3').text()
           if ((typeof caption !== 'string') || (caption.length === 0)) caption = $this.find(tpId).find('h4').text()
           if ((typeof caption !== 'string') || (caption.length === 0)) caption = $this.find(tpId).find('h5').text()
           if ((typeof caption !== 'string') || (caption.length === 0)) caption = $this.find(tpId).find('h6').text()
           if ((typeof caption !== 'string') || (caption.length === 0)) caption = "no title";
-          
+
 //          console.log("CAPTION: " + caption )
-          
+
           var tabName = document.createElement('span')
           tabName.setAttribute('class', 'sr-only')
           tabName.innerHTML='Slide ' + (i+1)
-          if (caption) tabName.innerHTML += ": " +  caption          
+          if (caption) tabName.innerHTML += ": " +  caption
           $tab.appendChild(tabName)
-          
+
          }
 
         // create div for focus styling of tablist
         $tablistHighlight = document.createElement('div')
         $tablistHighlight.className = 'carousel-tablist-highlight'
         document.body.appendChild($tablistHighlight)
-        
+
         // create button for screen reader users to stop rotation of carousel
 
         // create button for screen reader users to pause carousel for virtual mode review
         $complementaryLandmark = document.createElement('aside')
+        $complementaryLandmark.setAttribute('class', 'carousel-aside-pause')
         $complementaryLandmark.setAttribute('aria-label', 'carousel pause/play control')
-        $(document.body).prepend($complementaryLandmark)
-        
+        $this.prepend($complementaryLandmark)
```

Comment:
it seems like in [b9c43fb](https://github.com/paypal/bootstrap-accessibility-plugin/commit/b9c43fba29c50268723fc13e81d468b5c30f791b) made the change only in bootstrap-accessibility.js instead of the source carousel.js